src/legacy/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -73,20 +115,26 @@ export function GaugeChartProvider(Private) {
             const gauges = self.gauge.drawGauge(g, series, width, height);
 
             svg.attr('height', height);
+            //center the visualisation, Arc is centered differently, there's only
+            //a part of the circle visible and the circle is larger
             const transformX = width / 2;
-            const transformY = self.gaugeConfig.gaugeType === 'Arc' ? height / (2 * 0.75) : height / 2;
+            const transformY =
+              gaugeConfig.gaugeType === 'Arc'
+                ? height / (2 *  0.75)
+                : height / 2;
```

Comment:
```suggestion
const transformY = gaugeConfig.gaugeType === 'Arc' ? height / 2 + 20 : height / 2;
```
where 20 (can be also less) is the height of the title below the chart. In that way I see everything nicely center aligned